src/fragments/lib/auth/js/set-up-auth.mdx

```diff
@@ -269,11 +265,11 @@ If your existing UserPool client has a required attribute that is NOT set to mut
 </Block>
 </BlockSwitcher>
 
-**Recap:** You now have set up and configured your backend resources for Amplify authentication and are ready to start working on your front-end application.
+You now have set up and configured your backend resources for Amplify authentication and are ready to start working on your frontend application.
```

Comment:
You now have set up and configured your backend resources for Amplify Auth, and connected your frontend app to your backend. You are now ready to work on your frontend application.

Reply:
Updated